vinaigrette.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+import random
+
+
+def random_date_in_range():
+    date1 = input("Enter first date (YYYY-MM-DD):")
+    date2 = input("Enter second date (YYYY-MM-DD):")
+
+    try:
+        date1 = datetime.date.fromisoformat(date1)
+        date2 = datetime.date.fromisoformat(date2)
+    except ValueError:
```

Comment:
If you specify in the documentation or in the console output what the date format should be, you can just crush - python assumes the user is smart. If you want to leave this, throw a new exception with a custom message instead

Reply:
Apply the same thing on the other function